docs/concepts/index.rst

```diff
@@ -17,5 +17,4 @@ concepts that makes Hamilton unique and powerful.
    builder
    function-modifiers-advanced
    parallel-task
-   ui
```

Comment:
we'll just need to search for all the places we link to this.

E.g. blog, READMEs, Youtube I think.